### PR TITLE
Change default ping interval to 10s and make it configurable

### LINKED ISSUE #6: Make ping interval configurable
https://github.com/Konecty/meteor-multiple-instances-status/blob/master/multiple-instances-status.js#L3

We need to probably increase default, plus make it configurable.  Maybe environment variable?

### PR DESCRIPTION
**Pull request by sampaiodiego**
Closes #6

### CODE REVIEW

**Review by geekgonecrazy — APPROVED** [review]
I think this will make it much better!  👍

**Comment by sampaiodiego** [comment]
I guess the [index timeout](https://github.com/Konecty/meteor-multiple-instances-status/blob/master/multiple-instances-status.js#L7) needs to be changed too

**Comment by sampaiodiego** [comment]
@geekgonecrazy and/or @rodrigok can you please validate the new index creation logic and see if it makes sense? thx

File: multiple-instances-status.js [hunk 1]
```diff
@@ -1,10 +1,30 @@
 var events = new (Npm.require('events').EventEmitter)(),
 	collectionName = process.env.MULTIPLE_INSTANCES_COLLECTION_NAME || 'instances',
-	defaultPingInterval = 2000; // 2s
+	defaultPingInterval = (process.env.MULTIPLE_INSTANCES_PING_INTERVAL || 10) * 1000; // default to 10s
 
 var Intances = new Meteor.Collection(collectionName);
```
Comment (geekgonecrazy) [inline multiple-instances-status.js]: I know this isn't new.  :grin: but should be `instances`  Just noticed :grin:

Comment (sampaiodiego) [reply multiple-instances-status.js]: ohh.. a typo.. let me fix it

**Review by rodrigok — APPROVED** [review]

**Pull request merged**